net/xia/ppal_ether/main.c

```diff
@@ -497,6 +533,20 @@ static void free_neighs_by_interface(struct ether_interface *eint)
 	}
 }
 
+static void update_neighs_by_interface(struct ether_interface *eint)
+{
+	struct fib_xid_ether_main *mether;
+
+	ASSERT_RTNL();
+	rcu_read_lock();
+	list_for_each_entry_rcu(mether, &eint->list_interface_common_addr,
```

Comment:
Since we are under the RTNL lock, which `ASSERT_RTNL()` asserts, we don't need an RCU read lock. Under the  RTNL lock, one cannot add or remove an XID fib. Once the RCU read lock is dropped, use `list_for_each_entry()` instead of `list_for_each_entry_rcu`.